src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -98,6 +98,7 @@
          DelaySign="$(DelaySign)"
          DisabledWarnings="$(NoWarn)"
          DocumentationFile="@(DocFileItem)"
+         EmbedAllSources="$(EmbedAllSources)"
```

Comment:
Does this make sense doing it this way? For the MSBuild experience, I assume you'd just write:

```
<ItemGroup>
   <EmbeddedFiles Include="@(Compile)" />
</ItemGroup>
```

or something.